src/MoBi.Assets/AppConstants.cs

```diff
@@ -2068,6 +2071,11 @@ public static class PKSim
          public static readonly string PopulationSimulationArgument = "/pop";
          public static readonly string JournalFileArgument = "/j";
          public static readonly string NotInstalled = "PK-Sim was not found on current system. Please make sure that PK-Sim was installed using the provided setup. Alternatively, you can specify where PK-Sim is installed on your system under Utilities -> Options";
+
+         public static string EntryPointNotFound(string fileName)
+         {
+            return $"Could not find entry point {fileName}. Please make sure that PK-Sim was installed using the provided setup.";
```

Comment:
We have something like this where we try to start PKSim already (start population workflow). Maybe we could reuse the wording instead (it also specifies that path can be set explicitly in the user settings)